stellar/utils.ts

```diff
@@ -193,14 +204,13 @@ async function broadcast(operation, wallet, chain, action, options = {}, simulat
 
     if (simulateTransaction) {
```

Comment:
This feels out of place (the broadcast method looks like a bunch of independent logic being run depending on the action type, another example is the native payment). Create a separate simulate/query function?